src/database/migrations/20210724074759-modify_payments_add_field_comment.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.addColumn('Payments', 'comment', { 
+      type: Sequelize.STRING,
+      allowNull: true, 
+    });
+  },
+
+  down: async (queryInterface, Sequelize) => {
+    await queryInterface.removeColumn('Payments', 'comment');
+  },
+};
```

Comment:
удали пожалуйста неиспользованные параметры 
```suggestion
'use strict';

module.exports = {
  up: async (queryInterface, Sequelize) => {
    await queryInterface.addColumn('Payments', 'comment', {
      type: Sequelize.STRING,
      allowNull: true,
    });
  },

  down: async (queryInterface) => {
    await queryInterface.removeColumn('Payments', 'comment');
  },
};

```

Reply:
исправлено